Documentation/guides/Extending LiveCode.md

```diff
@@ -1250,7 +1257,7 @@ Additionally, all types can be annotated with **optional**. An optional annotati
         : 'module' <Name: Identifier> SEPARATOR
             { Metadata SEPARATOR }
             { Import SEPARATOR }
-            { Definition SEPARATOR }
```

Comment:
There's no difference between `use` and any other definition now, so you can get rid of the separate `Metadata` and `Import` lines and just include them in the option clause.